rustls/src/client/mod.rs

```diff
@@ -521,11 +527,14 @@ impl ClientSessionImpl {
     fn process_main_protocol(&mut self, msg: Message) -> Result<(), TLSError> {
         // For TLS1.2, outside of the handshake, send rejection alerts for
         // renegotation requests.  These can occur any time.
-        if msg.is_handshake_type(HandshakeType::HelloRequest) &&
+        /*if msg.is_handshake_type(HandshakeType::HelloRequest) &&
             !self.common.is_tls13() &&
```

Comment:
Est-ce qu'il ne faudrait pas supprimer uniquement le `&& !self.common.is_tls13()` et conserver les autres conditions pour rejeter la renegotiation ? Pourquoi ne plus rejeter dans aucun cas ?

Reply:
is_hanshaking at this step is false so we have to remove it also afterward the last if is not necessary :)